### ISSUE TITLE
hello,here is my question

### ISSUE THREAD

**Issue by HeiMay** [MingshanHe/Compliant-Control-and-Application#44]
I would like to ask whether <HybridAdmittance.launch> means hybrid impedance and admittance control

**Comment by MingshanHe** [comment]
> I would like to ask whether <HybridAdmittance.launch> means hybrid impedance and admittance control

No, this file use the admittance control algorithm in the z axis and position control in the x and y axis.  You could understand it as the force and position control easily.
